From 1c0c029576e6eed69be2f96b5bbcf13636ea3a84 Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Tue, 29 Mar 2016 14:17:27 +0500
Subject: [PATCH 1/2] disable tests

The tests subdirectory requires libva base package to
build which is disabled here.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index beac7e9..be1ce9e 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -24,7 +24,7 @@ ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
 
 AUTOMAKE_OPTIONS = foreign
 
-SUBDIRS = va dummy_drv_video pkgconfig test debian.upstream doc
+SUBDIRS = va dummy_drv_video pkgconfig debian.upstream doc
 
 # Extra clean files so that maintainer-clean removes *everything*
 MAINTAINERCLEANFILES = \
-- 
1.9.1

